packages/components-css/logo/index.scss

```diff
@@ -0,0 +1,19 @@
+.frameless-logo {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  inline-size: var(--frameless-size-2xl);
+  padding-block: var(--frameless-space-block-snail);
+  padding-inline: var(--frameless-space-inline-snail);
+}
```

Comment:
Je hebt nu geen component tokens gemaakt, zou dat niet handiger zijn? Bijvoorbeeld:

```css
  padding-block: var(--frameless-logo-padding-block);
```
Dan kun je de designkeuze in de design tokens doen.

In plaats van `inline-size`, zou je niet de design tokens voor `utrecht-logo` kunnen invullen?

Reply:
Component tokens gemaakt ✅